Lib/test/test_argparse.py

```diff
@@ -795,6 +795,13 @@ def test_invalid_name(self):
         self.assertEqual(str(cm.exception),
                          "invalid option name '--no-foo' for BooleanOptionalAction")
 
+    def test_prefix_chars_incompatibility(self):
+        parser = argparse.ArgumentParser(prefix_chars='+')
+        with self.assertRaises(ValueError) as cm:
```

Comment:
```suggestion
        with self.assertRaisesRegex(ValueError, ...) as cm:
```

no need to inspect the message after `with` :)